ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import os
+import platform
+
+from ros2pkg.api.create import create_folder
+from ros2pkg.api.create import create_template_file
+from ros2pkg.verb import VerbExtension
+
+
+class CreateVerb(VerbExtension):
+    """Create a new ROS2 package."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'package_name',
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.path.realpath(os.getcwd()),
+            help='Directory where to create the package')
+        parser.add_argument(
+            '--build-tool',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='Which build tool to use')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            default=getpass.getuser() + '@' + platform.uname()[1] + '.local',
```

Comment:
Using `.local` as a default is a bad idea imo. Based on experience too many users will keep it as is.